addons/binding/org.openhab.binding.netatmo/README.md

```diff
@@ -1,29 +1,30 @@
 # Netatmo Binding
```

Comment:
Can you exclude this file for now because of https://github.com/openhab/openhab2-addons/pull/2682

Reply:
Sure.  I will close this PR and create a new one.  I will omit the Netatmo Binding in the new PR, and I will fix the table errors above.